docs/dist/src/histogram-date-range.js

```diff
@@ -385,9 +420,15 @@ export let HistogramDateRange = class extends LitElement {
       return bar;
     });
   }
-  formatDate(rawDate) {
-    const date = dayjs(rawDate);
-    return date.isValid() ? date.format(this.dateFormat) : "";
+  formatDate(dateMS) {
+    if (Number.isNaN(dateMS)) {
+      return "";
+    }
+    const date = dayjs(dateMS);
+    if (date.year() < 1e3) {
```

Comment:
SUGGESTION: Maybe add a comment about the significance of this constant?

Reply:
sure, will do. also, this comment is referring to the generated file  (in the `docs` folder) that is built to be used for the github pages demo. all of the original source code is in the `src` and `test` folders